app/forms/declaration/NactCode.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration
+import forms.DeclarationPage
+import play.api.data.Forms.text
+import play.api.data.{Form, Forms}
+import play.api.libs.json.Json
+import services.NationalAdditionalCode
+import utils.validators.forms.FieldValidator._
+
+case class NactCode(nactCode: String)
+
+object NactCode extends DeclarationPage {
+
+  implicit val format = Json.format[NactCode]
+
+  val nactCodeKey = "nactCode"
```

Comment:
You don't need this key

Reply:
These keys are used as constants for field name in the view and tests